danceschool/vouchers/views.py

```diff
@@ -28,15 +28,15 @@ class GiftCertificateCustomizeView(FormView):
     template_name = 'cms/forms/display_form_classbased.html'
     form_class = VoucherCustomizationForm
 
-    def dispatch(self,request,*args,**kwargs):
+    def dispatch(self, request, *args, **kwargs):
         '''
         Check that a valid Invoice ID has been passed in session data,
         and that said invoice is marked as paid.
         '''
         paymentSession = request.session.get(INVOICE_VALIDATION_STR, {})
```

Comment:
paymentSession is also not PEP8-compliant :)

Reply:
Ah just to clarify should it be payment_session instead to follow https://www.python.org/dev/peps/pep-0008/#function-and-variable-names ?

Reply:
Yes